### DISCUSSION TITLE
看一看我国纪检监察机关的统计数据 | /project/ccdi/

### DISCUSSION THREAD

**Discussion by giscus** [earfanfan/yuanfan#630] [Announcements]
https://yuanfan.rbind.io/project/ccdi/

**Comment by botsp** [comment]
能不能再列一个每年在职的各级公务员人数，看下不同级别的犯罪率是多少，也和社会群众的犯罪率对比下，看看是人民群众当了官就能降低了犯罪率，还是公务员特别擅长把好人变成坏人、提高了犯罪率。

**Reply by earfanfan** [reply]
哇，朋友，你这是早上刚上班就来我这溜达了么，我以前搜过财政供养人员数量没搜到较为官方的结果，这个公务员职务犯罪和群众犯罪可能不大能直接比较。回头我搜搜看检察院、法院的官网上能不能找到一些相关数据吧。

**Comment by botsp** [comment]
看到更新就先打开拜读下，哈哈；就是感觉挺有意思的，可能现在数据不够、但若干年后肯定有做的，这几年新闻老是说抓了谁谁，都麻了，感觉都是坏人了。

**Reply by earfanfan** [reply]
我有关注半月谈、新华社、中国新闻周刊这类官方微信公众号，看多了官员被抓，倒没有觉得麻了，而是猜想这些信息放出来的目的是不是用来收拢民心的……前面提到的财政供养人员数量，我翻来翻去只翻到[这里](https://www.rmzxb.com.cn/c/2016-06-21/878388_1.shtml)：

>我国广义的财政供养人员到底是多少？根据国家统计局统计的数据，2008年政府机关人员1328万，事业单位人员2618万，财政供养人员合计为3946万人，占总人口的比重为2.97%，比值为1：34。据记者了解，2008年以后，国家统计局再没有对财政供养人员总数进行过统计，竹立家向《中国经济周刊》记者介绍，目前我国财政供养人员约5000万人。

粗略估算一下，财政供养人员5000万，每年被举报后立案处分的约60万人，比例大概就是1.2%，不过那些贪腐案件的涉案金额没有公布，一般公布信息只说涉案金额特别巨大。

最近看 B 站倒是比较多看到实名举报的。

**Comment by botsp** [comment]
不是有句口号吗, "反腐永远在路上"， 真替领导累得慌，一直跑、一直抓，老是在路上就不能歇歇吗。

**Reply by earfanfan** [reply]
哇，朋友，看你这样吐槽过这句话以后，我想我以后每次再看到“xx永远在路上”的口号也会忍不住替人家感到累了，哈哈哈。
